pkg/models/model_plan.go

```diff
@@ -21,6 +21,7 @@ func NewModelPlan(createdBy uuid.UUID, modelName string) *ModelPlan {
 	}
 
 }
+func (m ModelPlan) isLinkedExistingModel() {}
```

Comment:
Was this method meant to be filled?

Reply:
No, that is to represent the union type in GQL. It satisfies the interface